src/store/index.js

```diff
@@ -1,16 +1,33 @@
 import Vue from 'vue'
 import Vuex from 'vuex'
 
+import api from "@/api/FactoryRepository.js"
+const moviesApi = api.get("movies")
+
 Vue.use(Vuex)
 
 export default new Vuex.Store({
   state: {
+    movies: [],
   },
   getters: {
+    getMovies(state) {
```

Comment:
name `getMovies` suggests it is a method, but when using getters, they are used as values - you are not calling them as functions, right? for this reason it is good practice to name getters in a way that describes what value they represent e.g. `movies` or `movieList`